Assets/Scripts/SS3D/CodeGeneration/Creators/StaticClassCreator.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using UnityEditor;
+using UnityEngine;
+
+namespace SS3D.CodeGeneration
+{
+    public static class StaticClassCreator
+    {
+        #if UNITY_EDITOR
+        /// <summary>
+        /// Creates an enum in the object path, with the defined enum name and using a list of provided assets as its elements.
```

Comment:
is that an enum too ? Also looks a lot like DatabaseAssetCreator, maybe factor it ?